### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by Zeeerooo**
- Fix admin views
- Remove APScheduler, used uWSGI mules for background jobs.
- Listing reports directory
- Optimazed antennas geolocalization process
- Bug fixes
- Added test for events without sims

### CODE REVIEW

**Comment by sebablasko** [comment]
lgtm

File: app/automatization/scheduler_manager.py [hunk 1]
```diff
@@ -1,21 +1,29 @@
...
-def start_scheduler():
-    scheduler = BackgroundScheduler()
-    start_date = get_first_day()
-    scheduler.add_job(monthly_reports_generation, trigger="cron", id="Monthly reports generation",
-                      replace_existing=True,
-                      day="1", hour="0", minute="0", start_date=start_date)
-    scheduler.add_job(update_antennas_localization, kwargs={"max_number_of_queries": 1000}, trigger="cron",
-                      id="Antennas geolocalization", replace_existing=True, hour=3, minute=0)
-    scheduler.start()
+@cron(0, 0, 1, -1, -1, target="mule")
```
Comment (camilog) [inline app/automatization/scheduler_manager.py]: Add a little comment about what means this numbers

**Comment by camilog** [comment]
LGTM

**Comment by sebablasko** [comment]
lgtm

**Pull request merged**